README.md

```diff
@@ -23,7 +24,14 @@ replace({
   //Replacement to make
   replace: /Find\sme/g,
```

Comment:
Perhaps you could also be thinking about changing this to `replace: 'Find me'` since some might not understand this line, if you want me to include it in this PR, just tell me